arch/posix/include/posix_cheats.h

```diff
@@ -9,8 +9,8 @@
  * Header to be able to compile the Zephyr kernel on top of a POSIX OS
  */
 
-#ifndef _POSIX_CHEATS_H
-#define _POSIX_CHEATS_H
```

Comment:
You need to search and replace _POSIX_CHEATS_H in the CMake files to (that's why regression is failing)

Reply:
Or better, use that guard name but replace the 1st line
to `#if !defined(ZEPHYR_ARCH_POSIX_INCLUDE_POSIX_CHEATS_H_) && !defined(NO_POSIX_CHEATS)`
And replace in the CMake files `_POSIX_CHEATS_H` with `NO_POSIX_CHEATS`

Reply:
@aescolar thanks, I'll fix it

Reply:
should be fixed now...